pkg/skaffold/docker/image.go

```diff
@@ -122,7 +122,7 @@ func addTag(ref name.Reference, targetRef name.Reference, auth authn.Authenticat
 		return err
 	}
 
-	img, err := remote.Image(ref, auth, tr)
+	img, err := remote.Image(ref, remote.WithAuth(auth), remote.WithTransport(tr))
```

Comment:
was this meant to be changed?

Reply:
Yeah, this was a change I made in the go-containerregistry code that then got picked up here when I updated the version. Shouldn't have any effect.